generic_tracers/generic_COBALT_diag.yaml

```diff
@@ -0,0 +1,969 @@
+#
+# Limitation Diagnostics
+#
+- {name: '"P_C_max_Di"' , longname: '"Diaz. Maximum Growth Rate"'                   , hor_grid: "'h'", z_grid: "'L'", t_grid: "'s'", units: "'sec-1'"                    , mem_size: "'f'", var_to_register: "phyto(DIAZO )%id_P_C_max"   , axes: "axes(1:3)", missing_value: "missing_value1"}
```

Comment:
By checking the MOM6 code, I can confirm that a mix of quote styles (single vs. double quotes) exists on their side as well. @theresa-morrison, do you have any ideas for that?

Reply:
I think we should choose what ever is most readable here. If using only single or double quotes would mean we can eliminate having both  `' " x" '` and` " ' x ' "` that would be good -- I can see that tripping up a lot of users.

Reply:
Can we remove most of the quotes in the yaml file and have them inserted by the python code? I think we would just need to keep single quotes in the yaml file for strings with special characters like %.